packages/arb-ts/src/lib/l2Bridge.ts

```diff
@@ -36,7 +36,7 @@ const ARB_RETRYABLE_TX_ADDRESS = '0x000000000000000000000000000000000000006E'
 export interface L2TokenData {
   ERC20?: { contract: StandardArbERC20; balance: BigNumber }
   ERC777?: { contract: StandardArbERC777; balance: BigNumber }
-  CUSTOM?: { contract: IArbToken; balance: BigNumber } // Force custom to have l1Address (in, ie., IArbToken)
+  CUSTOM?: { contract: StandardArbERC20; balance: BigNumber } // Here we don't use the particlar custom token's interface; for the sake of this sdk that's fine
```

Comment:
I don't think its fine - this can lead to us making silly mistakes in the future. We should probably use the typechain class generated from the custom token interface - we can make it an abstract contract instead of interface.